config/sync/filter.format.basic_html.yml

```diff
@@ -16,7 +18,7 @@ filters:
     status: true
     weight: -10
     settings:
-      allowed_html: '<a href hreflang> <em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <p> <br> <span> <img src alt height width data-entity-type data-entity-uuid data-align data-caption> <panel>'
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <p> <br> <span> <img src alt height width data-entity-type data-entity-uuid data-align data-caption> <panel> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title>'
```

Comment:
Not quite sure what caused this to change.  Must have been one of the modules that got enabled.  I went through the change and it doesn't look like anything has actually been added or removed, just a slight re-arrangement.